packages/backend/unique-ids-generator/README.md

```diff
@@ -0,0 +1,21 @@
+<h1 align="center">Gravitywell Unique ids Generator</h1>
+<p align="center">Library for generating unique ids</p>
+<p align="center">
+  <img src="https://img.shields.io/github/workflow/status/GravitywellUK/packages/CI/master" alt="CI" />
+  <img src="https://img.shields.io/github/license/gravitywelluk/packages" alt="License" />
+  <img src="https://img.shields.io/npm/dm/@gravitywelluk/unique-ids-generator" alt="Downloads" />
+  <img src="https://img.shields.io/npm/v/@gravitywelluk/unique-ids-generator" alt="Version" />
+</p>
+<br />
+
```

Comment:
Can we add a description of what this generates by default?
For instance:
* Examples of the kind of ID that's generated
* A ballpark of _how_ random these are - e.g. how many different 4 character IDs are there?